### PR TITLE
PR for `fixes build error` 

### PR DESCRIPTION
**Pull request by MosesStewart**
In this issue we . . . 

- Removed the error in the `toml` file preventing people from downloading the package.

Adding @jmshapir

### CODE REVIEW

**Comment by MosesStewart** [comment]
I got the package downloaded but am having some trouble running the functions ~ still working

**Comment by MosesStewart** [comment]
@jmshapir I have the package working. Here is an example of what you can currently do with the package:
```
import BootstrapReport
import numpy as np
from scipy.stats import norm

rep_norm = BootstrapReport.generate_example_dataset(name = 'normal')
my_ooi = BootstrapReport.ObjectOfInterest(rep_norm[1], np.std(rep_norm[0], ddof = 1), rep_norm[0])
my_ooi.get_sk_min()
normal = norm(loc = 2, scale = 2)
sk_dis = BootstrapReport.helpers.get_sk_dist(rep_norm, normal, sep = False)
```
We specify which files inside of `src/BootstrapReport` that the user can call inside [`src/BootstrapReport/__init__.py`](https://github.com/JMSLab/BootstrapReport/blob/4598ac880f9163fe9a27528d9977842898dd7024/src/BootstrapReport/__init__.py). Right now I have `ObjectOfInterest`, `generate_example_dataset`, and all the functions in `helpers`. If we don't want the user to be able to easily call any of thoses, we can delete them.

I also renamed `BootstrapReport.py` to `report.py`. By default, python will name the package whatever the name of the folder is, and we can import the functions we want the user to be able to use in the `__init__` file.

File: src/BootstrapReport/examples/_generate_example_data.py
Comment (jmshapir) [inline src/BootstrapReport/examples/_generate_example_data.py:1]: I'd like to set this function to be non-user-callable. (It's an internal function just for our reference.)

File: pyproject.toml [hunk 1]
```diff
@@ -2,9 +2,6 @@
 requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools]
-packages = ["BootstrapReport"]
-
 [project]
 name = "BootstrapReport"
 version = "0.0.1"
```
Comment (jmshapir) [inline pyproject.toml:7]: I think we're now at [version 1.0.0](https://github.com/JMSLab/BootstrapReport/releases/tag/v1.0.0)?

File: tests/test_BootstrapReport.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from scipy.stats import norm, uniform
 from scipy.optimize import minimize
 import pytest
-from BootstrapReport.BootstrapReport import ObjectOfInterest
+from BootstrapReport.report import ObjectOfInterest
```
Comment (jmshapir) [inline tests/test_BootstrapReport.py:7]: Could something like this nomenclature work better? (Seems at least [somewhat conventional](https://stackoverflow.com/questions/37217786/is-there-a-set-name-i-need-to-give-my-py-file-containing-the-main-function), which is a plus.)

If you like it maybe we can rename `report.py`-->`main.py` and update throughout? 

```suggestion
from BootstrapReport.main import ObjectOfInterest
```

File: src/BootstrapReport/__init__.py [hunk 3]
```diff
@@ -1,7 +1,12 @@
+from .report import ObjectOfInterest
+from .examples._generate_example_data import generate_example_dataset
```
Comment (jmshapir) [inline src/BootstrapReport/__init__.py:2]: Let's not have `_generate_example_data` be user-accessible by default.

**Review by jmshapir — CHANGES_REQUESTED** [review]
@MosesStewart thanks!

I made a few suggestions throughout.

File: pyproject.toml (hunk 1 above)
Comment (MosesStewart) [reply pyproject.toml]: Addressed in 23e239b !

File: src/BootstrapReport/__init__.py (hunk 3 above)
Comment (MosesStewart) [reply src/BootstrapReport/__init__.py]: @jmshapir addressed in c2c4c72

File: src/BootstrapReport/examples/_generate_example_data.py
Comment (MosesStewart) [reply src/BootstrapReport/examples/_generate_example_data.py:1]: @jmshapir addressed in c2c4c72

**Review by jmshapir — APPROVED** [review]

File: src/BootstrapReport/__init__.py [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
     ============================
-    Use it to import very important functions.
     Package structure
     -----------------
     .
     ├── __init__.py
-    ├── examples -> Data calculations stored here
-        ├── __init__.py
-        ├── examples.py -> Examples file
     ├── BootstrapReport.py -> Main functions stored here
```
Comment (jmshapir) [inline src/BootstrapReport/__init__.py:13]:
```suggestion
    ├── main.py -> Main functions stored here
```

**Comment by jmshapir** [comment]
[Removed below from roadmap](https://github.com/JMSLab/BootstrapReport/wiki/Roadmap/_compare/110736a3b8994366e104508d7e4cd2bcc8a3ca5a...fc47967a7dc6aa70522665ece951f95ab33d00bf)

>   * Installation

**Pull request merged**